src/element/QueryTable/component/QueryTableParam.svelte

```diff
@@ -138,80 +111,69 @@
 		height: 0px;
 		margin-top: 4px;
 	}
-
 	[class^="param-"] {
 		align-self: center;
 		margin-bottom: 2px;
 	}
-
```

Comment:
oops - let's make sure to undelete this

Reply:
fixed line deletions in the most recent commit